### PR TITLE
Updated the Kahuna Kit to support Kahuna Android SDK 2.3.1

### PR DESCRIPTION
**Pull request by TorreyKahuna**

### CODE REVIEW

**Comment by TorreyKahuna** [comment]
@rayleeriver @sharekris please review.

File: kits/kahuna-kit/src/main/java/com/mparticle/kits/KahunaKit.java [hunk 1]
```diff
@@ -248,25 +253,43 @@ public void setUserIdentity(String id, MParticle.IdentityType identityType) {
...
+                break;
+            case Microsoft:
+                kahunaKey = "msft_id";
+                break;
+            case Yahoo:
+                kahunaKey = "yahoo_id";
+                break;
+            case Other:
+                kahunaKey = "mp_other_id";
                 break;
```
Comment (sharekris) [inline kits/kahuna-kit/src/main/java/com/mparticle/kits/KahunaKit.java:281]: Are these new keys fb_app_user_id, msft_id, yahoo_id, mp_other_id that our server will accept or convert them to attributes for now ?

**Comment by sharekris** [comment]
Looks good otherwise.

**Comment by rayleeriver** [comment]
LGTM

File: kits/kahuna-kit/src/main/java/com/mparticle/kits/KahunaKit.java (hunk 1 above)
Comment (TorreyKahuna) [inline kits/kahuna-kit/src/main/java/com/mparticle/kits/KahunaKit.java:281]: Right now they will be converted to attributes, but if we land a customer from mParticle who needs these tracked as User Credentials, we can consider adding them to our whitelist server side without going through another KahunaKit/mParticle SDK update.

**Pull request merged**